internal/pkg/skuba/addons/gangway.go

```diff
@@ -40,11 +41,8 @@ func (renderContext renderContext) GangwayImage() string {
 }
 
 func (renderContext renderContext) GangwayClientSecret() string {
-	client, err := kubernetes.GetAdminClientSet()
-	if err != nil {
-		return ""
-	}
-
+	// ignore error for `cluster init` case
+	client, _ := kubernetes.GetAdminClientSet()
```

Comment:
I would suggest to handle err, then if err exists, client = nil. 

Because sometimes we can not expect how the things will be handled in `kubernetes.GetAdminClientSet`to cause possible side effect.